src/component/menu/routes.js

```diff
@@ -350,6 +351,18 @@ export const routes = [
         menu: { mobile: true, advanced: true },
     },
 
+    // Segments
+
+    {
+        path: '/segments',
```

Comment:
I think we'll need a UI config flag to hide the segments pages for initial testing. Maybe `SE`?